Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -136,6 +136,13 @@ open class MapView: UIView {
         commonInit(mapInitOptions: mapInitOptions, overridingStyleURI: nil)
     }
 
+    /// :nodoc:
+    /// See https://developer.apple.com/forums/thread/650054 for context
+    @available(*, unavailable)
+    internal override init(frame: CGRect) {
+        fatalError("This initializer should not be called.")
```

Comment:
could we implement this instead using a default value of MapInitOptions?

Reply:
That's what we have in the initializer above. This is another swiftinterface issue it seems.

Reply:
interesting